src/sync/Synchroniser.js

```diff
@@ -111,9 +113,14 @@ export class Synchroniser {
           {
             headers: {
               Authorization: this.authHeader,
+              'msupply-site-uuid': uniqueId,
             },
           },
-        );
+        )
+          .then(response => response.json()
+            .then(responseJson => {
+              if (responseJson.error) throw new Error(responseJson.error);
```

Comment:
@raviSussol 

We've been using async, await instead of then(), catch etc..

in this case the code will look like this:
```
 const response = await fetch(
          `${this.serverURL}/sync/v3/initial_dump/` +
            `?from_site=${this.thisSiteId}&to_site=${this.serverId}`,
          {
            headers: {
              Authorization: this.authHeader,
              'msupply-site-uuid': uniqueId,
            },
          },
        );
        const json = await response.json();
        if(json.error && json.error != '')...
```

Reply:
No worries. Done.